### PR TITLE
[tests] fix Multi-Dex APK tests

### PR DESCRIPTION
**Pull request by jonathanpeppers**
It seemed this project didn't even build?

Things I fixed:

* `Mono.Android-Tests.Shared.projitems` had `\\`, `$(MSBuildThisFileDirectory)` has trailing slash
* Added `Image.9.png` to `Mono.Android-Tests.shared.projitems`
* `Mono.Android-TestsMultiDex.csproj` needed to adjust the location of `Resource.designer.cs`
* `Mono.Android-TestsMultiDex.csproj` was missing permissions
* `.gitignore` for `Resource.designer.cs`

### CODE REVIEW

**Comment by jonathanpeppers** [comment]
Test failure was: `JnienvTest.Mono.Android_TestsMultiDex, Java.InteropTests.JnienvTest.DoNotLeakWeakReferences / Release-Aot`, I think we've seen it randomly fail.

But it looks like this test project is running properly now:
https://jenkins.mono-project.com/job/xamarin-android-pr-builder-release/819/testReport/(root)/JnienvTest/

There are `Mono.Android_Tests` and `Mono.Android_TestsMultiDex`.

File: src/Mono.Android/Test/Mono.Android-Test.Shared.projitems [hunk 1]
```diff
@@ -9,41 +9,46 @@
...
+    <Compile Include="$(MSBuildThisFileDirectory)Xamarin.Android.RuntimeTests\NonJavaObject.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Xamarin.Android.RuntimeTests\NUnitInstrumentation.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Xamarin.Android.RuntimeTests\TestInstrumentation.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Android.OS\BundleTest.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Mono.Data.Sqlite\SqliteTests.cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <AndroidResource Include="$(MSBuildThisFileDirectory)..\..\Xamarin.Android.Build.Tasks\Tests\Xamarin.ProjectTools\Resources\Base\Image.9.png">
+      <Link>Resources\Drawable\Image.9.png</Link>
+    </AndroidResource>
```
Comment (dellis1972) [inline src/Mono.Android/Test/Mono.Android-Test.Shared.projitems]: If we are adding this here do we still need https://github.com/xamarin/xamarin-android/blob/2ba0f5f3cc0d30ce751df3122971c65ea443059e/tests/Runtime-MultiDex/Mono.Android-TestsMultiDex.csproj#L76 ?

File: tests/Runtime-MultiDex/Resources/Resource.designer.cs [hunk 2]
```diff
@@ -1,270 +0,0 @@
-#pragma warning disable 1591
```
Comment (jonpryor) [inline tests/Runtime-MultiDex/Resources/Resource.designer.cs]: Why is this file removed, when `Mono.Android-TestsMultiDex.csproj` was updated to add this file to `@(Compile)`?

File: src/Mono.Android/Test/Mono.Android-Test.Shared.projitems (hunk 1 above)
Comment (jonpryor) [reply src/Mono.Android/Test/Mono.Android-Test.Shared.projitems]: No, I wouldn't think that we would need that.

Comment (jonpryor) [reply src/Mono.Android/Test/Mono.Android-Test.Shared.projitems]: Removed.  Let's see if it still builds & runs! https://github.com/xamarin/xamarin-android/pull/2744/files#diff-53d0dd829aac4564989e7e33317ff2fcL74

**Comment by dellis1972** [comment]
Windows Failures are Designer Related and Lint? So not related to this. I'll merge.
